### ISSUE TITLE
Exit 1 Code. Can't find the problem mod(s)

### ISSUE THREAD

**Issue by Reignofmagic** [MultiMC/MultiMC5#1377]
Uploaded log

Let me know if you need any other relevant information Thanks in advance

http://paste.ee/p/viXAv

**Comment by peterix** [comment]
It's the bookshelf mod you have, or some other mod messed with the same thing before it did and it fails to apply its own changes. Hard to tell. But the mod is involved for sure.

**Comment by peterix** [comment]
Definitely two mods patching the same thing:
```
[17:58:55] [main/INFO] [AIC Core]: Patching Potion
```

```
Caused by: java.lang.ClassNotFoundException: net.minecraft.potion.Potion
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_66]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_66]
	at net.minecraft.potion.PotionHelper.<clinit>(SourceFile:82) ~[aen.class:?]
	at net.minecraft.item.Item.func_150900_l(Item.java:129) ~[adb.class:?]
	at net.minecraft.init.Bootstrap.func_151354_b(SourceFile:358) ~[kl.class:?]
	at net.minecraft.client.Minecraft.<init>(Minecraft.java:287) ~[bao.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:129) ~[Main.class:?]
	... 14 more
Caused by: net.darkhax.bookshelf.asm.ASMHelper$InvalidNeedleException: The needle was not found
	at net.darkhax.bookshelf.asm.ASMHelper.findLastNodeFromNeedle(ASMHelper.java:142) ~[Bookshelf-1.7.10-1.0.3.146.jar:?]
```

**Comment by Reignofmagic** [comment]
I saw that, just couldnt figure out which mods were messing with potions and causing the conflicts.

Ill try remocing the bookshelf api (new dependancy for telepads)

The issue already known conflict with anything that tracks potions. I just did not see that when updating my entire personal modpack today. 

When i get back on my computer Ill go back to an older version of telepads and let you know if the problem persists.
